index.js

```diff
@@ -26,7 +28,7 @@ function exportToJsonString(idbDatabase, cb) {
         } else {
           exportObject[storeName] = allObjects;
           if (
-            idbDatabase.objectStoreNames.length ===
+            objectStoreNames.length ===
```

Comment:
change this to objectStoreNames.size because a javascript Set don't have length in mobile devices